packages/app/src/cli/prompts/webhook/trigger.ts

```diff
@@ -1,19 +1,15 @@
 import {DELIVERY_METHOD, isAddressAllowedForDeliveryMethod} from '../../services/webhook/trigger-options.js'
 import {ui, output} from '@shopify/cli-kit'
 
-export async function topicPrompt(): Promise<string> {
+export async function topicPrompt(availableTopics: string[]): Promise<string> {
+  const choices = availableTopics.map((topic) => ({name: topic, value: topic}))
+
   const input = await ui.prompt([
```

Comment:
We are migrating the UI components, would you mind using [renderAutocompletePrompt](https://github.com/Shopify/cli/blob/27901edc9c91ca0323f7ba32dbb8f6ff9b55dbd4/packages/cli-kit/src/public/node/ui.tsx#L277) instead?